### PR TITLE
feat: Adding icon_url property for team.Application

### LINKED ISSUE #655: [REQUEST] Adding icon_url property for api.model.team.Application
### Describe the feature.

# **Describe the issue**
Currently, the only way to get the Application icon is manually getting the icon hash from ``icon`` and assign it into a link. My Pull Request will add a property method for doing this. For example, ``self.bot.me.icon_url``.

### Code of Conduct

- [X] I agree to follow the contribution requirements.

### PR DESCRIPTION
**Pull request by B1ue-Dev**
## About

This Pull Request adds a property method for getting the Application icon url in ``api.model.team.Application``.

## Checklist

- [x] I've ran `pre-commit` to format and lint the change(s) made.
- [x] I've checked to make sure the change(s) work on `3.8.6` and higher.
- [x] This fixes/solves an [Issue](https://github.com/goverfl0w/discord-interactions/issues).
  - (If existent): https://github.com/interactions-py/library/issues/655
- [x] I've made this pull request for/as: (check all that apply)
  - [ ] Documentation
  - [ ] Breaking change
  - [x] New feature/enhancement
  - [ ] Bugfix

### CODE REVIEW

File: interactions/api/models/team.py [hunk 1]
```diff
@@ -117,3 +117,19 @@ def __init__(self, **kwargs):
         self.owner = User(**self.owner) if self._json.get("owner") else None
         self.team = Team(**self.team) if self._json.get("team") else None
         self.flags = AppFlags(self.flags) if self._json.get("flags") else None
+
+    @property
+    def icon_url(self) -> str:
```
Comment (FayeDel) [inline interactions/api/models/team.py:122]: Add these to the .pyi headers as well.

**Comment by B1ue-Dev** [comment]
OK, I added those to the .pyi.

**Review by FayeDel — APPROVED** [review]
LGTM 👍🏻 
ty for contributing!

**Comment by B1ue-Dev** [comment]
> LGTM 👍🏻 ty for contributing!

No problem. Glad I can contribute something to make the library better.

**Pull request merged**